CUMULUS-2641 -- Update granule.productVolume to string/BigInt internal types (#2585)
nasa/cumulus: CHANGELOG.md

    present, the granule's last update date falls back to the `"Create"` type
    provider date, or `undefined`, if none is present.
- **CUMULUS-2311** - RDS Migration Epic Phase 2
  - **CUMULUS_2641**
    - Update API granule schema to set productVolume as a string value
    - Update `@cumulus/message` package to set productVolume as string

>       (calculated with BigIn from file.size) to match api schema

Reviewer: ```suggestion
      (calculated with `file.size` as a `BigInt`) to match API schema
```

>     - Update `@cumulus/db` granule translation methods to transalte updated

Reviewer: ```suggestion
    - Update `@cumulus/db` granule translation methods to translate updated
```
